test/Groups/matrixgroups.jl

```diff
@@ -143,6 +143,22 @@ end
    @test order(Oscar.isomorphic_group_over_finite_field(G)[1]) == 1
 end
 
+@testset "matrix group over QQBar" begin
+   #K = algebraic_closure(QQ)
+   K = Nemo.QQBar
```

Comment:
```suggestion
   K = algebraic_closure(QQ)
```
Nemo 0.41.3 already contains this. You just need to bump the compat entry in the `Project.toml`

Reply:
You changed the compat but didn't commit this change?

Reply:
indeed, forgot to `git add test` fixed now